modules/basic_topology/master.tf

```diff
@@ -105,3 +105,10 @@ resource "libvirt_domain" "ICP_boot" {
                private_key = "${file("${var.ssh_private_key_path}")}"
     }    
 }
+<<<<<<< HEAD
```

Comment:
Resolve these conflict. I expect no file is updated under "modules/basic_topology/"